platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -1440,10 +1551,12 @@ namespace platoon_strategic_ihp
          */
         bool isFrontJoin = plan_type.type == cav_msgs::PlanType::JOIN_PLATOON_FROM_FRONT;
         bool isRearJoin = plan_type.type == cav_msgs::PlanType::JOIN_PLATOON_AT_REAR;
-        bool isCutinJoin = plan_type.type == cav_msgs::PlanType::CUT_IN_FROM_SIDE;
+        bool isCutInJoin = plan_type.type == cav_msgs::PlanType::CUT_IN_FROM_SIDE;
+        bool isDepart = (plan_type.type == cav_msgs::PlanType::PLATOON_DEPARTURE);
+        // TODO: Place holder for departure plan type.
```

Comment:
This TODO should be removed.

Reply:
Indeed. The depart plan type is already included.